composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php"                   : ">=5.5",
+        "php"                   : "^5.5.9",
```

Comment:
Hello @dzuelke ; 

I'm working on a better support between blackfire and heroku.

I would like to avoid this king of patch, because it prevents users/travis to use PHP7.
I had to to that because our probe is not PHP7 compatible yet ; So I need a 5.X PHP version.
Is there a work-around for this?